schematic/models/metadata.py

```diff
@@ -178,34 +181,13 @@ def validateModelManifest(self, manifestPath: str, rootNode: str, jsonSchema: st
         # get annotations from manifest (array of json annotations corresponding to manifest rows)
         manifest = pd.read_csv(manifestPath).fillna("")
 
-
```

Comment:
@sujaypatil96 could you PR into 'develop'; we can review there - I want to make sure we are not removing code logic.